cmd/getroughtime/main.go

```diff
@@ -27,28 +27,27 @@ import (
 	"github.com/cloudflare/roughtime/config"
 )
 
-var (
+const (
 	// Build info.
 	Version   = "dev"
-	GoVersion = runtime.Version()
 	BuildTime = ""
-
-	// Command-line arguments.
-	getVersion = flag.Bool("version", false, "Print the version and exit.")
-	configFile = flag.String("config", "", "A list of Roughtime servers.")
-	pingAddr   = flag.String("ping", "", "Send a UDP request, e.g., localhost:2002.")
-	pingPubKey = flag.String("pubkey", "", "The Ed25519 public key of the address to ping.")
-	attempts   = flag.Int("attempts", client.DefaultQueryAttempts, "Number of times to try each server.")
-	timeout    = flag.Duration("timeout", client.DefaultQueryTimeout, "Amount of time to wait for each request.")
 )
 
 func main() {
+	// Command-line arguments.
+	getVersion := flag.Bool("version", false, "Print the version and exit.")
+	configFile := flag.String("config", "", "A list of Roughtime servers.")
+	pingAddr := flag.String("ping", "", "Send a UDP request, e.g., localhost:2002.")
+	pingPubKey := flag.String("pubkey", "", "The Ed25519 public key of the address to ping.")
+	attempts := flag.Int("attempts", client.DefaultQueryAttempts, "Number of times to try each server.")
+	timeout := flag.Duration("timeout", client.DefaultQueryTimeout, "Amount of time to wait for each request.")
```

Comment:
Did the linter suggest this? This change seems fine but more opinionated than I would expect a linter to be.

Reply:
Here is a list of all golangci-lint available linter, one of those is the gochecknoglobals linter which got active locally. I can change those back. Maybe we should decide on a configuration to use (or just keep the defaults). Does Cloudflare use a common linter configuration across Go projects?

Reply:
Thanks for the explanation, I'm happy to take this suggestion.

We don't have a standard linter config, at least for open source projects.

Reply:
It is indeed an opinionated linter, hence it is not part of the default set of linters. Additionally one of the commands in this repo already followed this schema, so figured I might as apply it to the others as well. Let me know if you got any preference here.

Reply:
Happy to take your suggestion.

Reply:
we take most of the by-default golangci-lint linter configuration, unless it gets really picky. This is a good start, and configuration can change in the future.